test/nostrum/shard/dispatch_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule Nostrum.Shard.DispatchTest do
+  use ExUnit.Case, async: true
+
+  alias Nostrum.Shard.Dispatch
+  alias Nostrum.Gateway.MessageDeleteEvent
+
+  describe "handle_event/1" do
+    test "returns `%MessageDeleteEvent{}` given Message Delete Event" do
+      result =
+        Dispatch.handle_event(
+          :MESSAGE_DELETE,
+          %{
```

Comment:
In the future we may want to use a payload like this multiple times. We should consider making this more accessible. It would also allow us to pretty easily throw some real examples into the mix if we wanted to make the tests more robust.

Reply:
Resolved in 7c53040. Let me know if I misunderstood the review.